Build.Dockerfile

```diff
@@ -1,10 +1,10 @@
 FROM ubuntu:18.04
 
-ARG ANDROID_COMPILE_SDK='29'
-ARG ANDROID_BUILD_TOOLS='28.0.3'
-ARG ANDROID_SDK_TOOLS='6200805'
-ARG OPEN_JDK_DOWNLOAD_URL='https://corretto.aws/downloads/resources/8.292.10.1/amazon-corretto-8.292.10.1-linux-x64.tar.gz'
-ARG OPEN_JDK_MD5='9d711fdeb9176a96bae0ba276f3f3695'
+ARG ANDROID_COMPILE_SDK='30'
+ARG ANDROID_BUILD_TOOLS='30.0.2'
+ARG ANDROID_SDK_TOOLS='7302050'
+ARG OPEN_JDK_DOWNLOAD_URL='https://corretto.aws/downloads/latest/amazon-corretto-11-x64-linux-jdk.tar.gz'
```

Comment:
Try not to use latest in general with docker. It means that next time you build an image, it might be new package already and behaviour will not be as you might expect.
You can grab version specific download link here https://github.com/corretto/corretto-11/releases

Reply:
oops, good catch, will change